phivk/gradu.al
# Fix: Corrects title for MozFest session cards.

To test:
- Go to a MozFest session and make sure it has MozFest in the title
- Go to a StoryTellers united  session and ensure it has StoryTellers United in the title.
